src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -50,33 +54,29 @@ export default function NurseryWithdrawals(props: NurseryWithdrawalsProps): JSX.
   const theme = useTheme();
   const contentRef = useRef(null);
   const classes = useStyles();
-  const [nurseryWithdrawals, setNurseryWithdrawals] = useState<SearchResponseElement[]>();
-
   const [searchResults, setSearchResults] = useState<SearchResponseElement[] | null>();
   const history = useHistory();
   const [searchValue, setSearchValue] = useState('');
   const debouncedSearchTerm = useDebounce(searchValue, 250);
   const [filters, setFilters] = useForm<NurseryWithdrawalsFiltersType>({});
-
-  useEffect(() => {
-    const getNurseryWithdrawals = async () => {
-      if (organization) {
-        const withdrawals = await listNurseryWithdrawals(organization.id, []);
-        if (withdrawals) {
-          setNurseryWithdrawals(withdrawals);
-        }
-      }
-    };
-
-    getNurseryWithdrawals();
-  }, [organization]);
+  const [species, setSpecies] = useState<Species[]>();
 
   const onWithdrawalClicked = (withdrawal: any) => {
     history.push({
       pathname: APP_PATHS.NURSERY_REASSIGNMENT.replace(':deliveryId', withdrawal.delivery_id),
     });
   };
 
+  useEffect(() => {
+    const populateSpecies = async () => {
+      const result = await getAllSpecies(organization.id);
+      if (result.requestSucceeded) {
+        setSpecies(result.species);
+      }
```

Comment:
should we show an error otherwise?

Reply:
Yes, we could show an error, but the thing is that you could still load the withdrawals and search if this fails, just the species filter would not have species to select if this fails. Showing an error explaining that wouldn't be confusing?